site/docs/v1/tech/apis/_application-response-body-base.adoc

```diff
@@ -339,6 +339,12 @@ The issuer that identifies the service provider and allows FusionAuth to load th
 [field]#{base_field_name}.samlv2Configuration.keyId# [type]#[UUID]# [since]#Available since 1.6.0#::
 The unique Id of the Key used to sign the SAML response.
 
+[field]#{base_field_name}.samlv2Configuration.loginHintConfiguration.enabled# [type]#[Boolean]# [since]#Available since 1.47.0#::
+Determines if support for a login hint sent by a SAML service provider is enabled for this application.
+
+[field]#{base_field_name}.samlv2Configuration.loginHintConfiguration.parameterName# [type]#[String]# [optional]#Optional# [default]#Defaults to `login_hint`# [since]#Available since 1.47.0#::
```

Comment:
```suggestion
[field]#{base_field_name}.samlv2Configuration.loginHintConfiguration.parameterName# [type]#[String]# [since]#Available since 1.47.0#::
```

Response doesn't need optional/required or default, because that doesn't make sense to document in a response.